src/animators/easingAnimator.ts

```diff
@@ -72,7 +72,7 @@ module Plottable.Animators {
      * @param {number} startDelay The start delay in milliseconds.
      * @returns {Easing} The calling Easing Animator.
      */
-    public startDelay(startDelay: number): Easing;
+    public startDelay(startDelay: number): this;
     public startDelay(startDelay?: number): any {
```

Comment:
This last signature is the *implementation*, and has to be assignable to any of the previous signatures (called "overloads"), according to the spec:

>When a function has overload declarations, the overloads determine the call signatures of the type given to the function object and the function implementation signature (if any) must be assignable to that type.

https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#61-function-declarations

The problem is that `number | this` isn't assignable to either `number` or `this` according to the casting rules, so you'll get a compiler error ("`Overload signature is not compatible with function implementation.`"). We can't simply leave out the `any` either, because that causes the error "`No best common type exists among return expressions.`" Typescript's [handbook](http://www.typescriptlang.org/Handbook#functions-overloads) includes an example where the implementation signature returns `any`.